### PR TITLE
vmui: add Active Queries page

### PR DESCRIPTION
**Pull request by Loori-R**
- Added a new page in VMUI to display active queries. 
- The API method `/api/v1/status/active_queries` is used to fetch the data.
- The API method `/api/v1/status/active_queries` has been updated; now it returns data in `JSON` format.

#4598

### CODE REVIEW

File: app/vmselect/promql/active_queries.go [hunk 1]
```diff
@@ -18,11 +18,23 @@ func WriteActiveQueries(w io.Writer) {
...
+            "query":%q,
+            "start":%d,
+            "end":%d,
+            "step":%d}`,
+            d.Seconds(), aqe.qid, aqe.quotedRemoteAddr, aqe.q, aqe.start, aqe.end, aqe.step)
+        if i+1 < len(aqes) {
+            fmt.Fprintf(w, `,`)
+        }
+    }
+    fmt.Fprintf(w, `]}`)
```
Comment (dmitryk-dk) [inline app/vmselect/promql/active_queries.go:37]: can you format this part of code?

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request!

**Comment by valyala** [comment]
@Loori-R , could you make the following UX enhancements to the `active queries` page in the follow-up pull request?

- To remove the `id` column, since it has zero value to the user.
- To join `from`, `to` and `step` columns into a single `args` column with the following template: `[<from> .. <to>], step=<step>`. For example, `[2023-01-05T10:24:43Z .. 2023-01-06T10:00:00Z], step=15s`.
- To rename `remote_addr` column to `client address`.
- To show columns in the following order:
  - `duration`
  - `client address`
  - `query`
  - `args`

The table must be sorted by descending order of `duration` column.

**Pull request merged**